spec/specification.yaml

```diff
@@ -0,0 +1,223 @@
+openapi: 3.0.2
+info:
+  title: GDPR Records
+  description: Exposes a basic service which hols the records of GDPR processing activities
+  version: '1.0'
+  termsOfService: 'https://fellesdatakatalog.brreg.no/about'
+  contact:
+    name: Brønnøysundregistrene
+    url: 'https://fellesdatakatalog.brreg.no'
+    email: fellesdatakatalog@brreg.no
+  license:
+    name: License of API
+    url: 'http://data.norge.no/nlod/no/2.0'
+externalDocs:
+  description: Dokumentasjon på Informasjonsforvaltning GitHub
+  url: https://informasjonsforvaltning.github.io/felles-datakatalog/datasett-katalog/api/
+tags:
+  - name: record
+    description: GDPR Record of Processing Activities
+servers:
+  - url: 'https://fellesdatakatalog.brreg.no/api'
+paths:
+  /records:
+    post:
+      tags:
+        - record
+      summary: Create a new data source
+      operationId: createRecord
+      responses:
+        '201':
+          description: Created
+          headers:
+            Location:
+              schema:
+                type: string
+              description: Relative location to the new data source
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Record'
+    get:
+      tags:
+        - record
+      summary: Query for GDPR records.
+      description: Returns a collection of matching GDPR records
+      operationId: getRecords
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Record'
+  /records/{id}:
+    get:
+      tags:
+        - record
+      description: Get a specific GDPR record by id
+      operationId: getRecordById
+      parameters:
+        - in: path
+          name: id
+          description: id
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Record'
+    put:
+      tags:
+        - record
+      description: Patch a specific GDPR record by id
```

Comment:
Description should reflect the operation. 'Patch' designates a different operation. Btw, should we use PATCH instead of PUT?